h2o-r/h2o-package/R/connection.R

```diff
@@ -348,7 +348,7 @@ h2o.connect <- function(ip = "localhost", port = 54321, strict_version_check = T
 h2o.getConnection <- function() {
   conn <- .attemptConnection()
   if (is.null(conn))
-    stop("No active connection to an H2O cluster. Did you run `h2o.init()` ?")
+    stop("No active connection to an H2O cluster. Did you run `SUBST_START_COMMAND` ?")
```

Comment:
I think this will definitely work but in my opinion, this is not a very robust solution. I think it would be better if we make it work without modifying files.

I suggest the following solution:

Introduce simple boolean flag "is_client" (naming can be different), populate this to true/false based existence of client_info or similar file in the package (you can also generate a file client_info.r during the build). Use an if/else statement to print out the correct message. Defaults should be false -> the package should work normally if the gradle breaks. That is the most common use case.

This has the advantage that it allows us to introduce argument checks in the h2o.init function when it is actually used in client-only mode.

I think this is a far flexible solution.

Reply:
It is implemented. I wasn't sure about content of client_info.txt so I used dist-readme.txt